src/Makefile.am

```diff
@@ -12,7 +12,7 @@ bindfs_LDADD = ${fuse_LIBS} ${fuse3_LIBS} ${fuse_t_LIBS} ${my_LDFLAGS}
 man_MANS = bindfs.1
 
 if BUILD_OS_IS_DARWIN
-bindfs_BUNDLEDIR = /Library/Filesystems/bindfs.fs
+bindfs_BUNDLEDIR = $(DESTDIR)/Library/Filesystems/bindfs.fs
```

Comment:
Where does this come from? Is it empty by default?

Reply:
@mpartel it's a convention used by e.g. Automake — https://www.gnu.org/software/automake/manual/html_node/DESTDIR.html — but not only to redirect files to a temporary location while keeping file system structure intact. For example in Gentoo, all(?) Makefile-based projects are installed using `DESTDIR` and then from there files are copied into the real system location in a later stage. I believe this tiny `Makefile` does a good job at demonstrating `DESTDIR`: https://github.com/tenox7/ttyplot/blob/71efd87c9b6629f524e9fed20a11742a17cba498/Makefile . And then in packaging it's used like this: https://github.com/gentoo/gentoo/blob/e31c0485124e5a80829765f715ad2370c4128cd9/app-admin/ttyplot/ttyplot-1.5.2.ebuild#L18-L23 . Does that help?

Reply:
@mpartel PS: yes, empty by default.

Reply:
Just felt weird seeing a prefix for MacOS `/Library`, since I don't recall ever seeing it in /usr or /usr/local etc. Anyway, probably fine if empty by default :+1: - thanks!